docs/contributing.rst

```diff
@@ -192,18 +189,13 @@ Making a new release
 
 A checklist for creating, testing, and distributing a new version.
 
-#. Choose a version number, e.g. "1.15.0"
-
-#. Update the changelog:
-
-   #. Add missing changes to :file:`docs/changelog.rst`.
-   #. Add a release line at the beginning referencing the release
-      and the date of the release.
-   #. Commit, push, ensure Travis build passes.
-
-#. Create a new git tag with ``git tag -m tag {number}``.
-#. Push the new tag: ``git push upstream {number}``.
-#. Watch the release `in Travis <https://travis-ci.org/pypa/twine>`_.
+#. Add missing changes to :file:`docs/changelog.rst`.
+#. Choose a version number, e.g. ``3.2.0``.
```

Comment:
I felt like updating the changelog would inform picking the version number.

Reply:
If we are following SemVer (I'm not sure if we've explicitly stated we are?) then breaking the changes into sections by version number, and then subsections of "Changed", "Added", "Fixed" would inform what version number to choose based on the presence of items in each section:

* "Changed" -> major version bump
* "Added" -> minor version bump
* "Fixed" -> patch version bump

Although I'm not sure how well that works with the current workflow.

Reply:
I don't think that's explicitly stated, but implicitly enforced by how `releases` manages `changelog.rst`: https://releases.readthedocs.io/en/latest/concepts.html (which is why I included it at the top of the file).